jwt.js

```diff
@@ -350,21 +379,31 @@ function fastifyJwt (fastify, options, next) {
     steed.waterfall([
       function getSecret (callback) {
         const verifyResult = secretCallbackVerify(request, decodedToken, callback)
-
         if (verifyResult && typeof verifyResult.then === 'function') {
           verifyResult.then(result => callback(null, result), callback)
         }
       },
       function verify (secretOrPublicKey, callback) {
-        jwt.verify(token, secretOrPublicKey, options.verify || options, (err, result) => {
-          if (err instanceof jwt.TokenExpiredError) {
+        try {
+          const verifierOptions = mergeOptionsWithKey(options.verify || options, secretOrPublicKey)
+          const verifier = createVerifier(verifierOptions)
```

Comment:
We should not be creating a verifier every time. It's expensive and remove most of the benefits of fast-jwt.